lblod/frontend-form-builder
# SFB-73/optimize codelijsten

## ID
 [SFB-73](https://binnenland.atlassian.net/browse/SFB-73)

 ## Description

The users do not understand how they have to add options to there dropdown component. They have to add a json configuration in a textarea to apply options for that dropdown.

I added a new tab in the topbar which is called `configuration` the idea behind this is that you can find all the extra configuration on sections/fields. Now I added a selector for the concept-scheme-uri's if the displaytype of the field is a selector component. They are listed in the code which ones.

With the idea of adding layout to fields/sections this can be added in this tab to.

Adding the previous branch here just as a reference for the code https://github.com/lblod/frontend-form-builder/pull/68

 ## Type of change


 ## How to test

Build a small form with minimum two sections (this is just for display) and a field that is a selector (eg. radiobuttons,selector)  in the builder. Once you did this you can add extra configuration to the fields that can hold `form:options`. Switch between the options for a field and look if the previous options are correctly applied and the old onces replaced. The preview should also refresh just as the code is updated in when switching to the code tab.

 ## Links to other PR's

- /
